scss/detail.scss

```diff
@@ -0,0 +1,89 @@
+@import 'colors';
+
+.rectangle {
+  background-color: $white-primary;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.rectangle-1 {
+  display: flex;
+  height: 416px;
+  width: 900px;
+  background-color: $white-secondary;
+  box-shadow: 4px 4px 10px 0 $swadow;
+}
+
+.container-img {
+  padding: 24px
+}
+
+.img {
+  height: 368px;
+  width: 261px;
+}
+
+.container-book {
+  display: flex;
+  height: 100%;
+  width: 100%;
+  padding: 24px;
+  flex-direction: column;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  border-bottom: solid $green-lemon;
```

Comment:
Just in case don't forget to specify the "border-width" with a width in px or another unit